ios/FluentUI/Core/Theme/Tokens/ConfigurableTokenizedControl.swift

```diff
@@ -29,8 +26,4 @@ protocol ConfigurableTokenizedControl: TokenizedControlInternal {
 /// This set of extensions implements the `TokenizedControlInternal` APIs for SwiftUI components.
 extension ConfigurableTokenizedControl {
     var overrideTokens: TokenType? { state.overrideTokens }
```

Comment:
After all these changes is there any benefit on still keeping the override tokens in the state object?

Reply:
Yeah, the main reason is for clients to be able to specify override tokens by adding them to the state object either directly (using our UIKit wrapper) or through the view modifier.

Reply:
I'm wondering if there's a way to keep all the logic of token management in the state object (through what the TokenizedControlInternal functionality accomplishes).

Reply:
Discussed offline: we'll continue to think about how to bring tokens together, but for now we'll merge this PR to unblock other work.